<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>150</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Youtube.png</normaloff>
    <normalon>Youtube.png</normalon>Youtube.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="linkLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>URL link:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="destinyLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>17</number>
    </property>
    <property name="text">
     <string>File Destination:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinyText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>425</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="folderButton">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>80</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>folder-open.png</normaloff>folder-open.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>115</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="videoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>118</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="audioButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>118</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Audio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="apagaVideoBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>118</y>
      <width>95</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar VÃ­deo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>115</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
